mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulVisitedProfileScreen.kt

```diff
@@ -25,6 +25,7 @@ class FetchedUserProfileScreenState(
     user: Profile,
     actions: State<ProfileActions>,
     chessFacade: ChessFacade,
+    val onBackToSocialClick: () -> Unit,
```

Comment:
You should introduce a hierarchy of `ProfileActions`, and make the new callback a member of one of the hierarchy members.

For instance :

```kotlin
interface ProfileActions {
  fun onNavigateToMatch(matchIdentifier: String)
}

interfaces VisitedProfileActions : ProfileActions {
  fun onBack()
}
```

Reply:
👍